c_glib/test/test-decimal128.rb

```diff
@@ -106,4 +106,81 @@ def test_divide_zero
       decimal1.divide(decimal2)
     end
   end
+
+  def test_not_equal
```

Comment:
Could you also add a test for `equal`?
(I didn't add it. Sorry.)

Reply:
Yes. I'll add it.